blueocean-dashboard/src/main/js/components/karaoke/components/Pipeline.jsx

```diff
@@ -114,13 +126,7 @@ export default class Pipeline extends Component {
                 logger.debug('Need to set new Run. Happens when e.g. re-run.');
                 nextProps.augmenter.setRun(nextProps.run);
             }
-            debounce(() => {
-                if (KaraokeConfig.getPreference('runDetails.pipeline.karaoke').value !== 'never') {
-                    logger.debug('re-setting karaoke mode.');
-                    this.karaoke = true;
-                }
-                this.pager.fetchNodes({ node: nextProps.params.node });
-            }, 200)();
```

Comment:
@bitwiseman same problem as described in: https://github.com/jenkinsci/blueocean-plugin/pull/2104
there is other misuse of debounce in this file
but this one specific cause the flooding of GET request